dev-docs/bidders/eightPod.md

```diff
@@ -11,10 +11,11 @@ gpp_sids: none
 schain_supported: false
 dchain_supported: false
 userId: none
-media_types: banner
+media_types: banner, video
 deals_supported: false
+userIds: check with bidder
 floors_supported: false
-fpd_supported: false
+fpd_supported: true
```

Comment:
sorry - you can't claim FPD support unless the document has a description of what kind of First Party data. From http://lh.prebid.org:8080/dev-docs/bidder-adaptor.html#submitting-your-adapter "If you support first party data, you must document what exactly is supported and then you may set fpd_supported: true. No default value."

See how other adapters have done this. Site first party data, user first party data, impression-level data, seller-define audience, etc.

Reply:
@GreDiSe - please review this comment

Reply:
@GreDiSe - see above